### ISSUE TITLE
Synonyms not working for address points

### ISSUE THREAD

**Issue by jmarin** [cfpb/grasshopper-loader#146]
The following query doesn't work (but if you replace Street by St it does). The same type of query does work on the census index. 

```json
POST /address/_search
{
    "query": {
        
   "bool" : {
     "must" : [ {
       "match" : {
         "state" : {
           "query" : "AR",
           "type" : "boolean"
         }
       }
     }, {
       "matchPhrase" : {
         "properties.street" : {
           "query" : "Main Street",
           "type" : "phrase"
         }
       }
     } ]
   }
    }
}
```

**Comment by hkeeler** [comment]
This _should_ be fixed with PR #144.  Once we reload the data and give it a whirl, we can close this one out.

**Comment by wpears** [comment]
Yep fixed. Thanks for the report
